msx/bank1/boot_menu.asm

```diff
@@ -1318,12 +1318,50 @@ BM_PRINT_CUR_DIR:
     ld hl,BM_DOTS_BAR_S
     call PRINT
 
+    ; Skip leading slash in the last directory part, then print it
     call BM_GET_LAST_DIR_PNT
-    call PRINT
-    jr _BM_PRINT_CUR_DIR_END
+    jr _BM_PRINT_CUR_DIR_TRUNC
+
+    ;call BM_GET_LAST_DIR_PNT
+    ;call PRINT
+    ;jr _BM_PRINT_CUR_DIR_END
```

Comment:
If this code is no longer used it's better to just remove it, and the same for the `;call PRINT` below.